kube/app/ingress-external.yml

```diff
@@ -15,7 +15,7 @@ spec:
         - {{ .DRONE_BUILD_NUMBER }}.{{ .BRANCH_ENV }}.homeoffice.gov.uk
         - {{ .APP_NAME }}-{{ .DRONE_SOURCE_BRANCH }}.{{ .BRANCH_ENV }}.homeoffice.gov.uk
       {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
-        - rotm.uat.internal.sas-notprod.homeoffice.gov.uk
+        - dev.notprod.{{ .APP_NAME }}.homeoffice.gov.uk
```

Comment:
Changing this isn't going to do anything but you are right it shouldn't have 'internal' in it. Maybe make this:
```
uat.notprod.{{ .APP_NAME }}.homeoffice.gov.uk
```